File: Facebook.NET/Contracts/GraphAPI.SecuritySettings.cs
using System;
using System.Runtime.Serialization;

namespace Facebook
{
    public static partial class GraphApi
    {
        [DataContract]
        public sealed class SecuritySettings
        {
            [DataMember(Name = "secure_browsing")]
            public Settings SecureBrowsing { get; set; }
        }
    }
}

File: Facebook.NET - VB/Contracts/GraphAPI.SecuritySettings.vb
Imports System
Imports System.Runtime.Serialization

Namespace Facebook
    Public NotInheritable Partial Class GraphApi
        <DataContract()>
        Public NotInheritable Class SecuritySettings
            <DataMember(Name:="secure_browsing")>
            Public Property SecureBrowsing() As Settings
        End Class
    End Class
End Namespace
